in-app-tutorials/getting-started/shared-tutorials/execute-with-kre.md

```diff
@@ -0,0 +1,55 @@
+## Execute tests with Katalon Runtime Engine
+
+1. Download Katalon Runtime Engine from Katalon website: [Katalon](https://katalon.com/download).
+
+2. Open the Command Prompt on your machine:
+   <details>
+      <summary>For macOS</summary>
+  
+     1. Extract the zip file.
+     2. In the extracted folder, right-click on **Katalon Studio Engine** and select "Show Package Contents".
+     3. Go to Contents > MacOS. Right-click on MacOS folder and select "New Terminal at Folder".
+
+   </details>
+
+   <details>
+      <summary>For Windows</summary>
+
+      1. Extract the zip file.
+      2. In the extracted folder, type “cmd” in the folder path and enter to launch the Command Prompt.
+
+   </details>
+
+   <details>
+      <summary>For Linux</summary>
+
+      1. Extract the zip file.
+      2. Right-click on the extracted folder and select "Open in Terminal". 
+
+   </details>
+
+3. Open **Command Builder** from the main toolbar.
+   <img src="https://github.com/katalon-studio/docs-images/raw/master/katalon-studio/in-app-tutorials/shared-images/open-command-builder.png" width=250>
+
+4. In the displayed **Generate Command for Console Mode** dialog, configure your execution with the provided options:
+
+   <img src="https://github.com/katalon-studio/docs-images/raw/master/katalon-studio/in-app-tutorials/shared-images/generate-command-console-mode.png" width=300>
+
+* **Test Suite:** select the Test Suite / Test Suite Collection you want to execute.
+
+* **Executive Platform:**
+   * **Run with:** Click on Edit and select an environment to run your tests with.
+
+* **Authentication:** the Katalon API key is auto-filled.
+
+5. Click **Generate Command** after completing the configuration.
+
+6. Click **Copy to Clipboard** and paste the command to your Command Prompt/Terminal for execution.
+
+   <img src="https://github.com/katalon-studio/docs-images/raw/master/katalon-studio/in-app-tutorials/shared-images/generated-command.png" width=280>
+
+7. [Click here](https://docs.katalon.com/docs/general-information/supported-integration/supported-integrations-in-katalon-platform#cicd-integration) to explore the supported CI/CD integrations.
```

Comment:
instead of having 7 steps, we should only have 6 steps, add a conclusion, and specify some example of CI/CD pipeline that users can integrate in that conclusion